resources/virt.go

```diff
@@ -1168,15 +1169,25 @@ func isNotFound(err error) bool {
 	return false // some other error
 }
 
-// expandHome does a simple expansion of the tilde into your $HOME value.
+// expandHome does an expansion of ~/ or ~james/ into user's home dir value.
 func expandHome(p string) (string, error) {
-	// TODO: this doesn't match strings of the form: ~james/...
-	if !strings.HasPrefix(p, "~/") {
-		return p, nil
+	if strings.HasPrefix(p, "~/") {
+		usr, err := user.Current()
+		if err != nil {
+			return p, fmt.Errorf("can't expand ~ into home directory")
+		}
+		return path.Join(usr.HomeDir, p[len("~/"):]), nil
 	}
-	usr, err := user.Current()
-	if err != nil {
-		return p, fmt.Errorf("can't expand ~ into home directory")
+
+	r, _ := regexp.Compile("~([^/]+)/")
```

Comment:
Do we need to check this error? I'd recommend putting a short comment above the regexp explaining the logic if you'd like.

More of a nitpick really, but overall patch looks great, thanks!